sqlite3.go

```diff
@@ -1039,7 +1045,7 @@ func (rc *SQLiteRows) Next(dest []driver.Value) error {
 		case C.SQLITE_INTEGER:
 			val := int64(C.sqlite3_column_int64(rc.s.s, C.int(i)))
 			switch rc.decltype[i] {
-			case "timestamp", "datetime", "date":
+			case timestamp, datetime, date:
```

Comment:
Is this correct?

Reply:
yes, I defined them as constant upstairs.

Reply:
done! PTAL